source/man/doveadm-search-query.7.rst.in

```diff
@@ -0,0 +1,300 @@
+
+**SENTON** *date specification*
+   Matches messages with a Date: header matching the given *date
+   specification*.
+
+**SENTSINCE** *date specification*
+   Matches messages with a Date: header matching or after the given
+   *date specification*.
+
+**SINCE** *date specification*
+   Matches messages whose internal date is within or after the given
+   *date specification*.
+
+**SMALLER** *size*
+   Matches messages with a size smaller than the given *size*.
+
+**SUBJECT** *string*
+   Matches messages, which contain *string* in the SUBJECT field of the
+   message's IMAP envelope structure.
+
+**TEXT** *string*
+   Matches messages, which contain *string* in the message headers or
+   body.
+
+**TO** *string*
+   Matches messages, which contain *string* in the TO field of the
+   message's IMAP envelope structure.
+
+**UID** *sequence-set*
+   Matches messages with the given IMAP UID(s). See the **sequence-set**
+   description for details on it. For example **1:10,100:200,***
+   matches the UIDs from 1 to 10, 100 to 200 and also the last mail.
+
+**UNANSWERED**
+   Matches messages, which do not have the IMAP flag \\Answered set.
+
+**UNDELETED**
+   Matches messages, which do not have the IMAP flag \\Deleted set.
+
+**UNDRAFT**
+   Matches messages, which do not have the IMAP flag \\Draft set.
+
+**UNFLAGGED**
+   Matches messages, which do not have the IMAP flag \\Flagged set.
+
+**UNKEYWORD** *keyword*
+   Matches messages, which do not have the given IMAP *keyword* flag set
+
+**UNSEEN**
+   Matches messages, which do not have the IMAP flag \\Seen set.
+
+DOVEADM SEARCH KEYS
+-------------------
+
+Additional search keys, provided by :man:`doveadm(1)`.
+
+**SAVEDBEFORE** *date specification*
+   Matches messages, which were saved before *date specification*.
+
+**SAVEDON** *date specification*
+   Matches messages whose save date matches the given *date
+   specification*.
+
+**SAVEDSINCE** *date specification*
+   Matches messages with a save date matching or after the given *date
+   specification*.
+
+DATE SPECIFICATION
+==================
+
+:man:`doveadm(1)` supports a few additional *date specification* formats.
+They can be used anywhere, where a *date specification* value is
+obligatory.
+
+*day* **-** *month* **-** *year*
+   Default IMAP4rev1 date format.
+
+   *day*
+      the day of month: **1**-**31**.
+
+   *month*
+      the abbreviated month name: **Jan**, **Feb**, **Mar**, **Apr**, **May**, **Jun**, **Jul**, **Aug**, **Sep**, **Oct**, **Nov** or **Dec**.
+
+   *year*
+      four digits of year, e.g. **2007**.
+
+   For example the "13th of April 2007" will be represented as **13-Apr-2007**.
+
+*interval*
+   Combination of a positive integer *number* and a *time unit*.
+   Available *time units* are:
+
+   **weeks**
+     abbr: **w**
+
+   **days**
+     abbr: **d**
+
+   **hours**
+     abbr: **h**
+
+   **mins**
+     abbr: **m**
+
+   **secs**
+     abbr: **s**
```

Comment:
I feel like "abbr" does not need to be abbreviated.

Reply:
i suppose so, it was in original text though =)